stubs/click-default-group/click_default_group.pyi

```diff
@@ -1,30 +1,86 @@
-from _typeshed import Incomplete
-from collections.abc import Sequence
+from collections.abc import Callable, MutableMapping, Sequence
+from typing import Any, Final, Literal, overload
+from typing_extensions import deprecated
 
 import click
 
 __all__ = ["DefaultGroup"]
-__version__: str
+__version__: Final[str]
 
 class DefaultGroup(click.Group):
     ignore_unknown_options: bool
     default_cmd_name: str | None
     default_if_no_args: bool
-    def __init__(self, *args, **kwargs) -> None: ...
+    # type hints were taken from click lib
+    def __init__(
+        self,
+        name: str | None = None,
+        commands: MutableMapping[str, click.Command] | Sequence[click.Command] | None = None,
+        *,
+        ignore_unknown_options: Literal[True] | None = True,
+        default_cmd_name: str | None = None,
+        default_if_no_args: bool = False,
+        invoke_without_command: bool = False,
+        no_args_is_help: bool | None = None,
+        subcommand_metavar: str | None = None,
+        chain: bool = False,
+        result_callback: Callable[..., Any] | None = None,  # Any is specified in click lib, and next also
+        context_settings: MutableMapping[str, Any] | None = None,
```

Comment:
I'd just copy the comment to the next line. This is safer, in case something changes, and someone looking at `context_settings` might miss the comment in the line above.

```suggestion
        result_callback: Callable[..., Any] | None = None,  # Any is specified in click lib
        context_settings: MutableMapping[str, Any] | None = None,  # Any is specified in click lib
```